apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -3426,6 +3429,11 @@ static void addProviderSlice(
       // Certain types have specific systems
       if (type == C4BBOrganizationIdentifierType.NPI) {
         id.setSystem(TransformerConstants.CODING_NPI_US);
+        if (!npiOrgName.isEmpty()) {
+          provider.setName(npiOrgName.get());
+        } else {
+          provider.setName("UNKNOWN");
+        }
       }
```

Comment:
Is this a big enough deal to bother logging, if there is no OrgName, or is this expected sometimes?

Reply:
I can log it, as info.. the Story Acceptance criteria is to set it to unknown if not found,